### DISCUSSION TITLE
Command Line installation failed

### DISCUSSION THREAD

**Discussion by Fourmarketing360** [globaleaks/globaleaks-whistleblowing-software#4240] [General]
### What version of GlobaLeaks are you using?

5.015

### What browser(s) are you seeing the problem on?

Other

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

Trying to install Globaleaks from command line, it fails after quite some time at "Running: apt-get install globaleaks"

Heres the output:

`root@ubuntu:~# ./install-globaleaks.sh
Running the GlobaLeaks installation...
In case of failure please report encountered issues to the ticketing system at: https://github.com/globaleaks/globaleaks-whistleblowing-software/issues

Detected OS: Ubuntu - jammy
WARNING: The recommended up-to-date platforms are Debian 12 (Bookworm) and Ubuntu 24.04 (Noble)
WARNING: Use one of these platforms to ensure best stability and security
Do you wish to continue anyway? [y|n]?y
Running: "apt-get -y update"... SUCCESS
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
tzdata is already the newest version (2024a-0ubuntu0.22.04.1).
0 upgraded, 0 newly installed, 0 to remove and 100 not upgraded.

Current default time zone: 'Etc/UTC'
Local time is now:      Wed Oct 16 06:57:27 UTC 2024.
Universal Time is now:  Wed Oct 16 06:57:27 UTC 2024.

Running: "apt-get -y install gnupg net-tools software-properties-common wget"... SUCCESS
Adding GlobaLeaks PGP key to trusted APT keys
Updating GlobaLeaks apt source.list in /etc/apt/sources.list.d/globaleaks.list ...
Running: "apt-get update -y"... SUCCESS
Running: "apt-get install globaleaks -y"... FAIL
Ouch! The installation failed.
COMBINED STDOUT/STDERR OUTPUT OF FAILED COMMAND:
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  apparmor-utils geoip-database libevent-2.1-7 libgeoip1 python3-acme
  python3-apparmor python3-geoip python3-gnupg python3-greenlet python3-h2
  python3-hpack python3-hyperframe python3-josepy python3-libapparmor
  python3-nacl python3-priority python3-pyotp python3-requests-toolbelt
  python3-rfc3339 python3-sqlalchemy python3-sqlalchemy-ext python3-txtorcon
  tor tor-geoipdb torsocks
Suggested packages:
  vim-addon-manager geoip-bin python-acme-doc python-greenlet-dev
  python-greenlet-doc python-nacl-doc python-sqlalchemy-doc python3-fdb
  python3-pymssql python3-mysqldb python3-psycopg2 python3-asyncpg
  python3-aiosqlite mixmaster torbrowser-launcher socat nyx obfs4proxy
The following NEW packages will be installed:
  apparmor-utils geoip-database globaleaks libevent-2.1-7 libgeoip1
  python3-acme python3-apparmor python3-geoip python3-gnupg python3-greenlet
  python3-h2 python3-hpack python3-hyperframe python3-josepy
  python3-libapparmor python3-nacl python3-priority python3-pyotp
  python3-requests-toolbelt python3-rfc3339 python3-sqlalchemy
  python3-sqlalchemy-ext python3-txtorcon tor tor-geoipdb torsocks
0 upgraded, 26 newly installed, 0 to remove and 100 not upgraded.
Need to get 85.2 MB/93.0 MB of archives.
After this operation, 154 MB of additional disk space will be used.
Get:1 https://deb.globaleaks.org jammy/ globaleaks 5.0.17 [85.2 MB]
Ign:1 https://deb.globaleaks.org jammy/ globaleaks 5.0.17
Get:1 https://deb.globaleaks.org jammy/ globaleaks 5.0.17 [85.2 MB]
Ign:1 https://deb.globaleaks.org jammy/ globaleaks 5.0.17
Get:1 https://deb.globaleaks.org jammy/ globaleaks 5.0.17 [85.2 MB]
Ign:1 https://deb.globaleaks.org jammy/ globaleaks 5.0.17
Get:1 https://deb.globaleaks.org jammy/ globaleaks 5.0.17 [85.2 MB]
Err:1 https://deb.globaleaks.org jammy/ globaleaks 5.0.17
  Connection timed out [IP: 95.174.23.119 443]
E: Failed to fetch https://deb.globaleaks.org/jammy/globaleaks_5.0.17_all.deb  Connection timed out [IP: 95.174.23.119 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?`

How can i fix this?

**Comment by evilaliv3** [comment]
Thank you @Fourmarketing360

This seems quite strange. Maybe your firewall is preventing you to reach our serve?

**Comment by Fourmarketing360** [comment]
It seems it needed port 21 opened, it was closed by default on the VPS, i opened it and the script was able to finish succesfully!

Thanks for the help!
